spec/controllers/group_health_controller_spec.rb

```diff
@@ -75,43 +118,220 @@
+            get :camps, format: :json
+            json = JSON.parse(response.body)
+            expect(json['camps'].size).to eq(1)
+            expect(json['camps'][0]['name']).to eq(events(:be_camp).name)
+          end
+
+          it 'does paginate' do
+            Role.create(group: groups(:be), person: people(:bulei),
+                        type: Group::Kantonalverband::Sekretariat.sti_name)
+            get :participations, params: { size: 3 }, format: :json
+            json = JSON.parse(response.body)
+            expect(json['participations'].size).to eq(3)
+            get :participations, params: { page: 2, size: 3 }, format: :json
+            json = JSON.parse(response.body)
+            expect(json['participations'].size).to eq(1)
+          end
+
+          it 'exports census evaluations' do
+            get :census_evaluations, format: :json
+            json = JSON.parse(response.body)
+            expect(json['census_evaluations']['kantonalverbaende'].size).to eq(1)
+            kantonalverband_evaluation = json['census_evaluations']['kantonalverbaende'].first
+            expect(kantonalverband_evaluation['kantonalverband_id']).to eq(groups(:be).id)
+            expect(kantonalverband_evaluation['region_id']).to eq(groups(:bern).id)
+            expect(kantonalverband_evaluation['abteilung_id']).to eq(groups(:schekka).id)
+
+            expect(kantonalverband_evaluation.keys.size).to eq(6)
+            expect(kantonalverband_evaluation['f'].size).to eq(7)
+            expect(kantonalverband_evaluation['m'].size).to eq(7)
+            expect(kantonalverband_evaluation['f']['leiter']).to eq(3)
+            expect(kantonalverband_evaluation['m']['leiter']).to eq(5)
+            expect(kantonalverband_evaluation['f']['pfadis']).to eq(5)
+            expect(kantonalverband_evaluation['m']['pfadis']).to eq(6)
+
+            expect(kantonalverband_evaluation['total']['total']).to eq(19)
+            expect(kantonalverband_evaluation['total']['f']).to eq(8)
+            expect(kantonalverband_evaluation['total']['m']).to eq(11)
+          end
+
+          it 'exports census evaluations of given year' do
+            get :census_evaluations, params: { year: '2013' }, format: :json
+            json = JSON.parse(response.body)
+            expect(json['census_evaluations']['kantonalverbaende']).to be_empty
+          end
         end
 
-        it 'does paginate' do
-          Role.create(group: groups(:schekka), person: people(:bulei),
-                      type: Group::Abteilung::Sekretariat.sti_name)
-          get :participations, params: { size: 3 }, format: :json
-          json = JSON.parse(response.body)
-          expect(json['participations'].size).to eq(3)
-          get :participations, params: { page: 2, size: 3 }, format: :json
-          json = JSON.parse(response.body)
-          expect(json['participations'].size).to eq(2)
+        context 'bern' do
```

Comment:
Sind `:be` und `:bern` eine Region und ein Kantonalverband? Wenn ja, könntest du die Contexts "Abteilung", "Region" und "Kantonalverband" nennen? Dann wäre das etwas klarer beim Lesen